packages/gatsby-theme-basic/src/components/modularity-modules/TableModule.module.css

```diff
@@ -20,14 +25,21 @@
 }
 
 .head {
-  background: var(--table--head-background, #f4f4f5);
+  background: var(--table-head-background, #f4f4f5);
   color: var(--table-head-color, inherit);
   border-bottom: var(--table-head-border-bottom, 1px solid transparent);
   border-bottom-color: var(--table-head-border-bottom-color, #d4d4d8);
+  font-size: var(--table-head-font-size, inherit);
+  font-weight: var(--table-head-font-weight, inherit);
+  font-style: var(--table-head-font-style, inherit);
 }
 
 .th,
 .td {
+  background: var(--table-cell-background, #f4f4f5);
+  font-size: var(--table-cell-font-size, inherit);
+  font-weight: var(--table-cell-font-weight, inherit);
+  font-style: var(--table-cell-font-style, inherit);
```

Comment:
Should we also add `text-align: left` to `th`? And also a default `font-weight` that’s different from the `td`s. At least a variable for it.